pkg/provider/prefix/provider.go

```diff
@@ -296,11 +296,16 @@ func (p *ipv4PrefixProvider) CreateIPv4PrefixAndUpdatePool(job *worker.WarmPoolJ
 
 	if err != nil {
 		p.log.Error(err, "failed to create all/some of the IPv4 prefixes", "created resources", resources)
-		didSucceed = false
+
 		//TODO: This adds a dependency on EC2 API error. Refactor later.
 		if strings.HasPrefix(err.Error(), utils.InsufficientCidrBlocksReason) {
```

Comment:
can you make not retryable errors into a list? if we have the list in config package, we can extend the list for not retry errors?

Reply:
Which file do you suggest to put the list?

Reply:
Thanks for the suggestion, made the changes.